sed/io.py

```diff
@@ -1,12 +1,16 @@
+from msilib import sequence
```

Comment:
My installation does not know msilib out of the box. Include corresponding requirements.

Reply:
[msilib](https://docs.python.org/3/library/msilib.html#module-msilib) — Read and write Microsoft Installer files
???
Why msi files? Please explain.
Also: Deprecated since version 3.11: The [msilib](https://docs.python.org/3/library/msilib.html#module-msilib) module is deprecated (see [PEP 594](https://www.python.org/dev/peps/pep-0594) for details).

Reply:
Why not typing.Sequence?

Reply:
This is some autocompletition error. was intended to be typing.Sequence

Reply:
are you using the `pre-commit` hooks? Because that kind of mistakes are avoided by either `isort` or `black`

Reply:
yes, I am, but it didn't catch this one as it was being used. There was a function which expected a `sequence` type instead of a `Sequence` type. 
It was a sneaky bug which passed through our pre-commit guards!